apps/cms/composer.json

```diff
@@ -105,6 +105,9 @@
       "drupal/gutenberg": {
         "Gutenberg enabled hook": "https://www.drupal.org/files/issues/2024-05-07/gutenberg_enabled_hook_3445677-2.patch",
         "Remove !important from sidebar": "./patches/gutenberg_remove-important-sidebar.patch"
+      },
+      "drupal/graphql": {
+        "Check if translation exists when loading an entity by its uuid": "./patches/graphql_load_by_uuid_translation_check.patch"
```

Comment:
Is this a custom patch? Or does it exist on drupal.org ?

It would be good to follow the convention for Drupal patches:

`#<Drupal-Issue-Number> - <Drupal-Issue-Title>` such as few lines above would be changed from:

>`"Gutenberg enabled hook": "https://www.drupal.org/files/issues/2024-05-07/gutenberg_enabled_hook_3445677-2.patch",`

To:

>`"#3445677 - Gutenberg enabled hook": "https://www.drupal.org/files/issues/2024-05-07/gutenberg_enabled_hook_3445677-2.patch",`

It would also be nice if we can contribute back these patches to drupal.org in a new issue